packages/static/kairos-overlay-files/files/system/oem/02_agent.yaml

```diff
@@ -1,11 +1,11 @@
 name: "Start agent"
 stages:
   boot:
-    - if: '[ ! -f "/run/cos/recovery_mode" ] && [ -d "/usr/share/systemd" ]'
+    - if: '[ ! -f "/run/cos/recovery_mode" ] && [ "$(kairos-agent state get kairos.init)" == "systemd" ]'
```

Comment:
not for now, but seeing this, makes me feel like we could simply have some querying mechanism in the agent e.g.

```
kairos-agent state is kairos.init.systemd
echo $?
1
```

Reply:
umm, not sure. State is state right? do we want to introduce that kind of operations onto it? would be cool indeed and we could piggyback the existing state I guess, but seems like a can of worms hahaha